WordPress/src/main/java/org/wordpress/android/ui/reader/discover/ReaderCardUiState.kt

```diff
@@ -20,7 +20,7 @@ sealed class ReaderCardUiState {
         val blogId: Long,
         val dateLine: String,
         val title: UiString?,
-        val blogName: String?,
+        val blogName: UiString?,
```

Comment:
I'm wondering if we could make this field non-nullable now, wdyt?

Reply:
You're right, fixed in b40758d